modules/Factoids.py

```diff
@@ -2,7 +2,9 @@
 import random
 import sqlite3
 from typing import Optional
-
+from os import path
+from config import factoid_database_path
+assert isinstance(factoid_database_path, str) and path.dirname(factoid_database_path), f"Factoid.db parent directory not found at {factoid_database_path=}"
```

Comment:
if the assert fails during deploy from GH action, does the GH action itself fail / error gets logged into Discord / error is only silently reported in the secret server logs?

Reply:
I assumed `assert` from anywhere would kill Stampy, but now that I think about it, it might only kill the module. I don't know if that would be logged

Reply:
An assert in a module doesn't produce any message. Moved to `config.py` instead